packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -342,8 +342,32 @@ export class ActionBar
       expanded,
       overflowCount,
     });
+
+    this.removeOverflowGroups(height, actionGroups, layout);
   }, overflowActionsDebounceInMs);
 
+  private removeOverflowGroups(
```

Comment:
@josercarcamo can you elaborate on what this is doing in the PR body?

Reply:
@josercarcamo I thought it would just be up to the author to remove the actions that won't fit into their content. I wasn't aware we were going to hide actions or groups. Maybe we can discuss this one again?